deebotozmo/commands.py

```diff
@@ -29,6 +29,23 @@ def __init__(self, speed: str):
         })
 
 
+class SetScheduleOnce(Command):
+
+    def __init__(self, hour: str, minute: str ):
+        super().__init__("setSched", {
+            "act": "add",
+            "content": {"jsonStr":"{\"type\":\"auto\"}","name":"clean"},
+            "enable": 1,
+            "hour": int(hour),
+            "index": 2,
+            "mid":"1037360161",
+            "minute": int(minute),
+            "repeat":"0000000",
+            "sid":"16311188160301",
```

Comment:
Is this value random?

Reply:
sid? Sid is the total of milliseconds

Reply:
👍 Then please change it that always the actual time is used

Reply:
@ the moment i call the sid like this in command.py

```
class ScheduleBase(Command):

    def millis():
        return str(round(time_.time() * 1000))

    def __init__(self, hour: int, minute: int, sid = millis()):
        super().__init__("setSched", {
            "act": "add",
            "content": {"jsonStr":"{\"type\":\"auto\"}","name":"clean"},
            "enable": 1,
            "hour": hour,
            "index": 2,
            "mid":"1037360161",
            "minute": minute,
            "repeat":"0000000",
            "sid": sid,
            "trigger":"app"
        })
```

But as you mentioned, it is a library. When i look, better push

```
    def millis():
        return str(round(time_.time() * 1000))
```
to util.py But after that i get an importerror for import from deebotozmo.util import millis. 
I tried just to "import from deebotozmo.util import md5".
Mh, same result...

Reply:
probably circular dependencies